test-files/cmd/glacial/NetworkConflateCmdInputReverseTest/output.osm

```diff
@@ -5520,8 +5520,8 @@
         <nd ref="-753"/>
```

Comment:
This was the only test that got affected (also ran regression).  I only looked at the K St/US 29 change in this test so far, and it made sense to me, as the more of the road got joined together.